oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/parser/listener/MultiScopesAnalysisListener.java

```diff
@@ -390,6 +390,8 @@ private Layer fromSpanLayerValue(SpanLayer spanLayer) {
                 return Layer.CACHE;
             case UNRECOGNIZED:
                 return Layer.UNDEFINED;
+            case FAAS:
+                return Layer.FAAS;
```

Comment:
Need to return `Layer.GENERAL` by default? I'm not sure @wu-sheng @wankai123

Reply:
I don't think should return `Layer.GENERAL` by default. The Layer should be set specifically.